docs/content/filtering.md

```diff
@@ -39,52 +73,56 @@ only match a complete path element:
               - doesn't match "afile.jpg"
               - doesn't match "directory/file.jpg"
 
-**Important** Note that you must use `/` in patterns and not `\` even
-if running on Windows.
+**Important** Use `/` in path/file name patterns and not `\` even if
+running on Microsoft Windows.
 
-A `*` matches anything but not a `/`.
+Eg `*` matches anything but not a `/` separator:
 
     *.jpg  - matches "file.jpg"
            - matches "directory/file.jpg"
            - doesn't match "file.jpg/something"
 
-Use `**` to match anything, including slashes (`/`).
+Eg `**` matches anything, including slash (`/`) separators:
 
     dir/** - matches "dir/file.jpg"
            - matches "dir/dir1/dir2/file.jpg"
            - doesn't match "directory/file.jpg"
            - doesn't match "adir/file.jpg"
 
-A `?` matches any character except a slash `/`.
+Eg `rclone --exclude /dir1/** ls remote:` lists all of `remote`
+except for root directory `dir1` and all the files contained within it
+or its subdirectories.
+
+Eg `?` matches any character except a slash `/` separator:
 
     l?ss  - matches "less"
           - matches "lass"
           - doesn't match "floss"
 
-A `[` and `]` together make a character class, such as `[a-z]` or
-`[aeiou]` or `[[:alpha:]]`.  See the [go regexp
-docs](https://golang.org/pkg/regexp/syntax/) for more info on these.
+Eg `[` and `]` together make a character class, such as `[a-z]`, `[aeiou]`
```

Comment:
There are a lot of `Eg`s here... Do we need them?

Reply:
There are! I will take a close look.